index.js

```diff
@@ -137,7 +137,8 @@ var TYPES = {
 
   quacksLike: function quacksLike (type) {
     function _quacksLike (value) {
-      return type === getValueTypeName(value)
+      return getValueTypeName(new ManglingCanary()) === 'ManglingCanary' &&
```

Comment:
Could this be moved to outside `_quacksLike`,  but inside `quacksLike`?

Reply:
Also,  wouldn't you want `_quacksLike` to return `true` (aka, PASS) if the mangling canary failed?

Reply:
Yes, you are right, reordered the code a bit and fixed the logic